solidity/contracts/deposit/Deposit.sol

```diff
@@ -117,179 +185,69 @@ contract Deposit is DepositFactoryAuthority {
-    /// @dev                    The signers will be penalized if this is not called.
-    /// @param  _txVersion      Transaction version number (4-byte Little Endian).
-    /// @param  _txInputVector  All transaction inputs prepended by the number of inputs encoded as a VarInt, max 0xFC(252) inputs.
-    /// @param  _txOutputVector All transaction outputs prepended by the number of outputs encoded as a VarInt, max 0xFC(252) outputs.
-    /// @param  _txLocktime     Final 4 bytes of the transaction.
-    /// @param  _merkleProof    The merkle proof of inclusion of the tx in the bitcoin block.
-    /// @param  _txIndexInBlock The index of the tx in the Bitcoin block (0-indexed).
-    /// @param  _bitcoinHeaders An array of tightly-packed bitcoin headers.
-    function provideRedemptionProof(
-        bytes4 _txVersion,
-        bytes memory _txInputVector,
-        bytes memory _txOutputVector,
-        bytes4 _txLocktime,
-        bytes memory _merkleProof,
-        uint256 _txIndexInBlock,
-        bytes memory _bitcoinHeaders
-    ) public {
-        self.provideRedemptionProof(
-            _txVersion,
-            _txInputVector,
-            _txOutputVector,
-            _txLocktime,
-            _merkleProof,
-            _txIndexInBlock,
-            _bitcoinHeaders
-        );
-    }
-
-    /// @notice     Anyone may notify the contract that the signers have failed to produce a signature.
-    /// @dev        This is considered fraud, and is punished.
-    function notifySignatureTimeout() public {
-        self.notifySignatureTimeout();
+        return self.auctionValue();
     }
 
-    /// @notice     Anyone may notify the contract that the signers have failed to produce a redemption proof.
-    /// @dev        This is considered fraud, and is punished.
-    function notifyRedemptionProofTimeout() public {
-        self.notifyRedemptionProofTimeout();
+    /// @notice Get caller's ETH withdraw allowance.
+    /// @dev Generally ETH is only available to withdraw after the deposit
+    ///      reaches a closed state. The amount reported is for the sender, and
+    ///      can be withdrawn using `withdrawFunds` if the deposit is in an end
+    ///      state.
+    /// @return The withdraw allowance in wei.
+    function withdrawableAmount() public view returns (uint256) {
+        return self.getWithdrawAllowance();
     }
 
-    //
-    // FUNDING FLOW
-    //
+//------------------------------ FUNDING FLOW --------------------------------//
 
-    /// @notice     Anyone may notify the contract that signing group setup has timed out.
-    function notifySignerSetupFailure() public {
-        self.notifySignerSetupFailure();
+    /// @notice Anyone may notify the contract that signing group setup has
+    ///         timed out if retrieveSignerPubkey is not successfully called
+    ///         within the allotted time.
+    /// @dev This
+    ///      RegisteredPubkey event with the two components. Reverts if the
```

Comment:
not following this bit

Reply:
That's
Because I apparently completely messed it up 😆